tests/integration/devfile/cmd_devfile_describe_test.go

```diff
@@ -0,0 +1,71 @@
+package devfile
```

Comment:
Can you add a rule to the makefile so that the tests in here can be run individually?

Something like
```
.PHONY: test-cmd-devfile-describe
test-cmd-devfile-describe:
	ginkgo $(GINKGO_FLAGS) -focus="odo devfile describe command tests" tests/integration/devfile/
```

Reply:
Done.